Dockerfile

```diff
@@ -73,6 +76,7 @@ COPY --from=dependencies ${GEM_HOME} ${GEM_HOME}
 COPY --from=dependencies ${DEPS_HOME}/package.json ${APP_HOME}/package.json
 COPY --from=dependencies ${DEPS_HOME}/yarn.lock ${APP_HOME}/yarn.lock
 COPY --from=dependencies ${DEPS_HOME}/node_modules ${APP_HOME}/node_modules
+COPY --from=dependencies ${DEPS_HOME}/rollup.config.js ${APP_HOME}/rollup.config.js
```

Comment:
This works 👍  I think the template has prompted this line to be added here which is now odd given we have more copy commands further down.

I think this is fine to merge but in case it's useful, I think we should head towards a place where we copy dependency files and app files separately.

```
# Copy dependencies (relying on dependencies using the same base image as this)
COPY --from=dependencies ${GEM_HOME} ${GEM_HOME}
COPY --from=dependencies ${DEPS_HOME}/node_modules ${APP_HOME}/node_modules
# End

# Copy app code (sorted by vague frequency of change for caching)
RUN mkdir -p ${APP_HOME}/log
RUN mkdir -p ${APP_HOME}/tmp

COPY Gemfile ${APP_HOME}/Gemfile
COPY Gemfile.lock ${APP_HOME}/Gemfile.lock
COPY package.json ${APP_HOME}/package.json
COPY yarn.lock ${APP_HOME}/yarn.lock
COPY rollup.config.js ${APP_HOME}/rollup.config.js
COPY config.ru ${APP_HOME}/config.ru
COPY Rakefile ${APP_HOME}/Rakefile
COPY script ${APP_HOME}/script
COPY public ${APP_HOME}/public
COPY vendor ${APP_HOME}/vendor
COPY bin ${APP_HOME}/bin
COPY config ${APP_HOME}/config
COPY lib ${APP_HOME}/lib
COPY db ${APP_HOME}/db
COPY app ${APP_HOME}/app
```